configure.ac

```diff
@@ -1392,6 +1392,11 @@ if test "$Py_LTO" = 'true' ; then
           LLVM_AR_FOUND=found
           AC_MSG_NOTICE([llvm-ar found via xcrun: ${LLVM_AR}])
         fi
+        # /usr/bin/ar on macOS works fine in place of llvm-ar
```

Comment:
Here, you will just override the LLVM_AR variable, then the previous value was computed. You have to use a `else`.

Other question, could you check that `/usr/bin/ar` exists...

@ned-deily what's your opinion?